### PR TITLE
feat(browser-starfish): add resource summary page

### PR DESCRIPTION
**Pull request by DominikB2014**
When you click on a resource in the resources module, the following resource summary page will appear instead of the sidebar. 
image

This isn't the final page, we should additionally add a resource size info, but we will work off this.

The reason for adding this is:
1. To provide some consistency between db and resource module.
2. To provide some additional context aka location, on what pages a specific resource was loaded in. 
3. To provide a place to add more content if we get bundle level information.

### CODE REVIEW

**Review by edwardgou-sentry — APPROVED** [review]
Looks reasonable to me!

File: static/app/views/performance/browser/resources/resourceSummaryPage/resourceSummaryTable.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+  const renderBodyCell = (col: Column, row: Row) => {
+    const {key} = col;
+    if (key === 'spm()') {
+      return <ThroughputCell rate={row[key] * 60} unit={RateUnits.PER_SECOND} />;
+    }
+    if (key === 'avg(span.self_time)') {
+      return <DurationCell milliseconds={row[key]} />;
+    }
+    return <span>{row[key]}</span>;
```
Comment (AbhiPrasad) [inline static/app/views/performance/browser/resources/resourceSummaryPage/resourceSummaryTable.tsx:51]: m: should we make these links to a transaction like db queries page does it?

File: static/app/views/performance/browser/resources/resourceSummaryPage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+              <Block title={getThroughputTitle('http')}>
+                <ThroughputCell
+                  rate={spanMetrics?.[`${SpanFunction.SPM}()`] * 60}
+                  unit={RateUnits.PER_SECOND}
+                />
+              </Block>
+            </BlockContainer>
+          </HeaderContainer>
+          <DescriptionContainer>
+            <SpanDescription span={spanMetrics} />
```
Comment (AbhiPrasad) [inline static/app/views/performance/browser/resources/resourceSummaryPage/index.tsx:94]: l: should be bump up the font size of the description and make it bold? An `h3` might work well.

**Review by AbhiPrasad — APPROVED** [review]

File: static/app/views/performance/browser/resources/resourceSummaryPage/resourceSummaryTable.tsx (hunk 1 above)
Comment (DominikB2014) [reply static/app/views/performance/browser/resources/resourceSummaryPage/resourceSummaryTable.tsx:51]: I'm thinking of doing the same span samples sidebar just like db queries does? Any thoughts on that?

Comment (AbhiPrasad) [reply static/app/views/performance/browser/resources/resourceSummaryPage/resourceSummaryTable.tsx:51]: yeah works for me!

File: static/app/views/performance/browser/resources/resourceSummaryPage/index.tsx (hunk 2 above)
Comment (DominikB2014) [reply static/app/views/performance/browser/resources/resourceSummaryPage/index.tsx]: Good call, I changed it to an h3!

**Pull request merged**